docs/migrating-from-2.0.md

```diff
@@ -92,4 +92,4 @@ The `Period` enum has also been moved from `ChuckCollector` to `RetentionManager
 
 ## 5. registerDefaultCrashHanlder typo
 
-The function `Chuck.registerDefaultCrashHanlder` contained a typo in the name and now is moved to `Chucker.registerDefaultCrashHandler`.
\ No newline at end of file
+The function `Chuck.registerDefaultCrashHanlder` contained a typo in the name and now is moved to `Chucker.registerDefaultCrashHandler`.
```

Comment:
What's the change here?

Reply:
New like at the end of file change, so we can ignore this.